sunpy/util/sysinfo.py

```diff
@@ -12,30 +12,43 @@
 __all__ = ['system_info', 'find_dependencies', 'missing_dependencies_by_extra']
 
 
-def get_requirements(package):
+def get_requirements(package, names_only=False):
```

Comment:
I think we would always want to return this inflammation instead of making it a flag.

Reply:
Could you explain what you mean by not making it a flag?

Reply:
Sorry, I mean as a keyword argument.

Reply:
Thanks for clarifying. Just updated the function to remove `names_only`.

One more question - I noticed that some optional dependencies are listed under more than one extra group (e.g. `jplephem`, `matplotlib`). Do all group info need to be displayed in those cases?

Reply:
That is a good question, that might be nice to see that info. How hard would it be to add that?

Reply:
I think it can be added to `find_dependencies`, like a function that resolves / combines info if the same package shows up more than once. I can work on making that update!